<!--xml-->
<launch>
   <node name="$(env ROS_HOSTNAME)_sample_node" pkg="icarus_rover_v2" type="sample_node"  output="screen" clear_params="true">
	    <param name="verbosity_level" value="INFO"/>
        <param name="require_pps_to_start" value="true" />
  </node>
</launch>
